__tests__/any-of-labels.spec.ts

```diff
@@ -0,0 +1,106 @@
+import {Issue} from '../src/classes/issue';
+import {IssuesProcessor} from '../src/classes/issues-processor';
+import {IIssuesProcessorOptions} from '../src/interfaces/issues-processor-options';
+import {DefaultProcessorOptions} from './constants/default-processor-options';
+import {generateIssue} from './functions/generate-issue';
+
+describe('any-of-labels option', () => {
+  test('should do nothing when not set', async () => {
+    const sut = new IssuesProcessorBuilder()
+      .emptyAnyOfLabels()
+      .issues([{labels: [{name: 'some-label'}]}])
+      .build();
+
+    await sut.processIssues();
+
+    expect(sut.staleIssues).toHaveLength(1);
+  });
+
+  test('should skip it when none of the issue labels match', async () => {
+    const sut = new IssuesProcessorBuilder()
+      .anyOfLabels('skip-this-issue,and-this-one')
+      .issues([{labels: [{name: 'some-label'}, {name: 'some-other-label'}]}])
+      .build();
+
+    await sut.processIssues();
+
+    expect(sut.staleIssues).toHaveLength(0);
+  });
+
+  test('should skip it when the issue has no labels', async () => {
+    const sut = new IssuesProcessorBuilder()
+      .anyOfLabels('skip-this-issue,and-this-one')
+      .issues([{labels: []}])
+      .build();
+
+    await sut.processIssues();
+
+    expect(sut.staleIssues).toHaveLength(0);
+  });
+
+  test('should process it when one of the issue labels match', async () => {
+    const sut = new IssuesProcessorBuilder()
+      .anyOfLabels('skip-this-issue,and-this-one')
+      .issues([{labels: [{name: 'some-label'}, {name: 'skip-this-issue'}]}])
+      .build();
+
+    await sut.processIssues();
+
+    expect(sut.staleIssues).toHaveLength(1);
+  });
+
+  test('should process it when all the issue labels match', async () => {
+    const sut = new IssuesProcessorBuilder()
+      .anyOfLabels('skip-this-issue,and-this-one')
+      .issues([{labels: [{name: 'and-this-one'}, {name: 'skip-this-issue'}]}])
+      .build();
+
+    await sut.processIssues();
+
+    expect(sut.staleIssues).toHaveLength(1);
+  });
+});
+
+class IssuesProcessorBuilder {
+  private _options: IIssuesProcessorOptions;
+  private _issues: Issue[];
+
+  constructor() {
+    this._options = {...DefaultProcessorOptions};
+    this._issues = [];
+  }
+
+  anyOfLabels(labels: string): IssuesProcessorBuilder {
+    this._options.anyOfLabels = labels;
+    return this;
+  }
+
+  emptyAnyOfLabels(): IssuesProcessorBuilder {
+    return this.anyOfLabels('');
+  }
+
+  issues(issues: Partial<Issue>[]): IssuesProcessorBuilder {
+    this._issues = issues.map((issue, idx) =>
+      generateIssue(
+        this._options,
+        idx,
+        issue.title || 'Issue title',
+        issue.updated_at || '2000-01-01T00:00:00Z', // we only care about stale/expired issues here
```

Comment:
Using a spread operator would be cleaner IMO.
You can just merge the default values with the given issue.
What do you think?

Reply:
Sure, that could work. I saw you had that function to generate the mock issues and thought it would be frowned upon if I didn't use it lol. I'm a fan of fluent builder patterns  for this type of setup, maybe adding a `IssueBuilder` in the future can be useful 😉.